nucypher/policy/policies.py

```diff
@@ -411,11 +409,12 @@ def make_arrangement(self, ursula: Ursula, *args, **kwargs):
         raise NotImplementedError
 
     @abstractmethod
-    def sample_essential(self, quantity: int, handpicked_ursulas: Set[Ursula] = None) -> Set[Ursula]:
+    def sample_essential(self, quantity: int, handpicked_ursulas: Set[Ursula]) -> Set[Ursula]:
         raise NotImplementedError
 
-    def sample(self, handpicked_ursulas: Set[Ursula] = None) -> Set[Ursula]:
-        selected_ursulas = set(handpicked_ursulas) if handpicked_ursulas else set()
+    def sample(self, handpicked_ursulas: Optional[Set[Ursula]] = None) -> Set[Ursula]:
+        handpicked_ursulas = handpicked_ursulas or set()
+        selected_ursulas = set(handpicked_ursulas)
```

Comment:
Feels like the previous one-liner was simpler

Reply:
Fair enough, not sure how it ended up this way. Fixed.